build error on `main`

Filing this for visibility, probably due to upstream changes / changes in the branch we're pinned to in tendermint-rs?:

```
error[E0308]: mismatched types
   --> pd/src/app.rs:212:32
    |
212 |                 retain_height: 0,
    |                                ^ expected struct `Height`, found integer

error[E0308]: mismatched types
   --> pd/src/app.rs:139:17
    |
139 |                 last_block_height,
    |                 ^^^^^^^^^^^^^^^^^ expected struct `Height`, found `i64`
    |
note: return type inferred to be `Height` here
   --> pd/src/app.rs:128:66
    |
128 |             let (last_block_height, last_block_app_hash) = match state.latest_block_info().await? {
    |                                                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

--- a/pd/Cargo.toml
+++ b/pd/Cargo.toml
@@ -21,8 +21,8 @@ penumbra-wallet = { path = "../wallet" }
 ark-ff = { git = "https://github.com/penumbra-zone/algebra", branch = "ours" }
 decaf377 = { git = "https://github.com/penumbra-zone/decaf377" }
 tower-abci = { git = "https://github.com/penumbra-zone/tower-abci/" }
-tendermint-proto = { git = "https://github.com/penumbra-zone/tendermint-rs.git", branch = "abci-domain-types" }
-tendermint = { git = "https://github.com/penumbra-zone/tendermint-rs.git", branch = "abci-domain-types" }
+tendermint-proto = { git = "https://github.com/penumbra-zone/tendermint-rs.git", branch = "abci-domain-types_r12" }
+tendermint = { git = "https://github.com/penumbra-zone/tendermint-rs.git", branch = "abci-domain-types_r12" }
 # External dependencies
 async-stream = "0.2"
 bincode = "1.3.3"

